docs/netlify.toml

```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
 [[redirects]]
 from = "/docs/rasa/action-server/deploy-action-server"
 to = "/docs/rasa/deploy/deploy-action-server/"
+
+[[redirects]]
+from = "/docs/rasa-x/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
```

Comment:
not sure if this rule will work, it conflicts with [this one on rasa-website](https://github.com/RasaHQ/rasa-website/blob/111584e42f7a6ac8bd68dccc8a432c274c41c718/netlify.toml#L402-L404)